setup/gcp/installer/forseti_server_installer.py

```diff
@@ -47,25 +49,58 @@ def __init__(self, **kwargs):
         """
         super(ForsetiServerInstaller, self).__init__()
         self.config = ServerConfig(**kwargs)
+        self.v1_config = None
 
     def preflight_checks(self):
         """Pre-flight checks for server instance"""
 
         super(ForsetiServerInstaller, self).preflight_checks()
-
+        gcloud.enable_apis(self.config.dry_run)
+        forseti_v1_name = None
+        if not self.config.dry_run:
+            _, zone, forseti_v1_name = gcloud.get_vm_instance_info(
+                r'^forseti-security-\d+-vm$', try_match=True)
```

Comment:
You might want to make this pattern a global if you think you can reuse it elsewhere.

Reply:
Moved the pattern to util/constants file.